gallery/visualtests/testUtils.js

```diff
@@ -6,6 +6,7 @@
  */
 const puppeteer = require('puppeteer');
 
+// eslint-disable-next-line no-undef
```

Comment:
I assume adding the `node` env to the eslint config would be the more correct way to fix this